### DISCUSSION TITLE
How send and receive data trought cable

### DISCUSSION THREAD

**Discussion by zapewne** [egzumer/uv-k5-firmware-custom#568] [Announcements]
Hello.

I have a usb<->audio *2 io to quansheng. How I can send and receive data trought it? I need sending and receive digital data trought radio (no setup, only send data to other radio trought radio wave).

I need simplest way to send and receive data.

**Comment by dkxce** [comment]
You can use 'UZ7HO Sound Modem' as ADC/DAC and 'dg1faz Paxon' as gateway. But it works in analog mode, no usb i/o. You must use on-board sound card or usb sound card. UV-K5 have build-in BK4819 FSK modem, but question is to accessible it from outdoor software.

**Reply by fab1an2** [reply]
UKk5 have  to read the data on the multi-radio has its own digital converter. It can read and send data much better than a sound card. Besides, you require a whole computer. Why . When you can use a radio to do it using less power and be more mobile.

If you could read and send data directly using firmware like the T9 extension, but from a USB device, you could control it with an arduino or router or other device like a cell phone.

This is a very good question. But requiring a change in firmweare.

**Reply by zapewne** [reply]
>The question is, does the circuit design provide direct access to the DAC for usage it as modem, what 
>modulation will be and which speed?

This is the subject of the question. It seems that if the firmware allows such transmission then the speed will be variable from 600-2400 baud. 
And most importantly it will be easy to write a client of any network. 
There could also be interoperability between the world of ham radio and those without a license.

**Reply by dkxce** [reply]
You can ask Nunu (@kamilsss655) about their Mesh network.
https://github.com/kamilsss655/uv-k5-firmware-custom/wiki/43-%E2%80%90-Mesh-network

**Reply by fab1an2** [reply]
> You can ask Nunu (@kamilsss655)  

Good idea, ask him and wrote here answer
